src/mapgen/mapgen_fractal.cpp

```diff
@@ -1,7 +1,6 @@
 /*
 Minetest
-Copyright (C) 2015-2018 paramat
-Copyright (C) 2015-2018 kwolekr, Ryan Kwolek <kwolekr@minetest.net>
```

Comment:
Why did you remove this credit? Much was changed in this file, but there's surely a concept or even part of the old code left over.

Reply:
Ok, i wasn't sure anyway.
The entire mapgen was written by me, but indeed much is code copied form hmmmm's work.
Will re-add.

Reply:
Done.